policies/README.md

```diff
@@ -1 +1,47 @@
-This directory holds documents of policies for the Astropy Project.  For example, rules for who gets certain rights or permissions and what the expectations are if they do.
\ No newline at end of file
+# Astropy Policies
+
+This directory holds documents of policies for the Astropy Project.  For example, rules for who gets certain rights or permissions and what the expectations are if they do.
+
+## Committee/Team Specifics
+
+### Coordination Committee
+- Policies: [CoCo Operating Policies](coco-operating-policies.md)
+- Records: [Running Notes](https://docs.google.com/document/d/1t7gEE6gIgNUCyObQFq3aYFMcOh4V4m27CNrxBGYwQuw/edit)
+- Internal Communication
+  - Bi-weekly closed meetings on Project calendar
+  - Private slack channel
+- External Communication
+  - Email: coordinators@astropy.org
+  - Github team [@astropy/coordinators](https://github.com/orgs/astropy/teams/coordinators)
+  - Quarterly updates to email lists.
```

Comment:
I didn't know we write quarterly updates. Is that still a thing?

But I think we do present a "State of Astropy" at Astropy Coordination Meetings.

Reply:
Ha yes, I wasn't sure about this either. I am guessing it's not?